tests/Functional/Configuration/PassThroughTest.php

```diff
@@ -186,8 +186,6 @@ private function getPossibleFutureOptions(): array
     {
         return [
             '--exclude-filter',
-            '--exclude-testsuite',
-            '--test-suffix',
```

Comment:
Why are you removing this from the passthrough? This could mean a breaking change for a user that has that options already in use it.

Reply:
Because this test gone fail Tests\Functional\Configuration\PassThroughTest::testAvoidOverlaps. 

What do you suggest to do?

Reply:
You should try to move both options under the `allowedOptionsDataProvider`, because it was missing. In this way, we cover both usages.

Reply:
done